CHANGELOG.md

```diff
@@ -7,8 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
 
+Fix RT alignment of libraries by swapping x and y coordinates of linear regression
+Option to select irts from 1st and 4th RT quantile only to avoid overfitting to the center of the RT distribution
```

Comment:
ah interesting. did you find that problem or did someone suggest that?

Reply:
1) So the x and y problem I found out about today and its luckily a small change
2) The quantile option I tried while figuring out what was the problem in the RT alignment and I think it might be usefull in some cases - so I kept it as an option